.gitignore

```diff
@@ -1,5 +1,6 @@
 # Object files
 *.o
+!backing.o
```

Comment:
Why are you adding backing.o and backing .img into the repo?  Binary things are bad in version control.